planetarium/NineChronicles
# Update ranking board query.

### Description

1. Update ranking board query to optimize client.
2. Open ability ranking.

### How to test

1. Check if ability ranking is open and it's working properly.
2. Check if other ranking categories are still working properly.

### Related Links
https://app.asana.com/0/958521740385861/1201690646326974

### Required Reviewers
@planetarium/9c-dev @Namyujeong
